head/firmware/main.cpp

```diff
@@ -185,6 +191,15 @@ auto main() -> int {
     head_tasks::start_tasks(can_bus_1, motor_left.motion_controller,
                             motor_left.driver, motor_right.motion_controller,
                             motor_right.driver, psd);
+    static auto timer_for_notifier =
```

Comment:
having this be an in-function `static` will prevent it from being overwritten by freertos, but why have it be an in-function `static` as opposed to just be defined outside a function at all.